src/app/find-help/reply-offer/reply-offer.component.html

```diff
@@ -70,14 +70,26 @@ <h3>{{ "REPLY_OFFER" | translate }}</h3>
           name="consent"
         >
           <span>{{ "ACCEPT" | translate }} </span>
-          <a target="_blank" href=""
-            ><span>{{ "CONSENT" | translate }}</span></a
-          >
-          <span *ngIf="!authService.isLoggedIn()">
-            <span> {{ "AND" | translate }} </span>
-            <a target="_blank" href="https://pomagamukrainie.gov.pl/assets/rodo/politykaprywatnosci.pdf"
-              ><span>{{ "PRIVATE_POLICY" | translate }}</span></a
-            >
+          <span *ngIf="currentLang === LanguageCode.pl_PL">
+            <a target="_blank" href="https://pomagamukrainie.gov.pl/assets/rodo/politykaprywatnosci.pdf">
+              <span>{{ "PRIVATE_POLICY" | translate }}</span>
+            </a>
+          </span>
+          <span *ngIf="currentLang === LanguageCode.uk_UA">
+            <a target="_blank" href="https://pomagamukrainie.gov.pl/assets/rodo/politykaprywatnosci_UK.pdf">
+              <span>{{ "PRIVATE_POLICY" | translate }}</span>
+            </a>
+          </span>
+          <span *ngIf="currentLang === LanguageCode.en_GB">
+            <a target="_blank" href="https://pomagamukrainie.gov.pl/assets/rodo/politykaprywatnosci_EN.pdf">
+              <span>{{ "PRIVATE_POLICY" | translate }}</span>
+            </a>
+          </span>
+          <span *ngIf="currentLang === LanguageCode.ru_RU">
+            <--! TODO update private policy with RU ver -->
+            <a target="_blank" href="https://pomagamukrainie.gov.pl/assets/rodo/politykaprywatnosci_EN.pdf">
+              <span>{{ "PRIVATE_POLICY" | translate }}</span>
+            </a>
```

Comment:
Can we instead set the `href` in the component code and not repeat whole blocks here?

BTW, the key should be named `PRIVACY_POLICY` but that's already there.

Reply:
or
<a target="_blank" [href]=" 'RODO_LINK' | translate ">
  <span>{{ "PRIVATE_POLICY" | translate }}</span>
</a>
but the link will be in the translation key

Reply:
That seems like abusing the translation system, though. :)

Reply:
How about having a reusable snippet (component?) covering this scenario in reply-offer.component.html and site-footer.component.html (see #197 )

Reply:
@ermenegildo500 that seems like the best option, indeed.

Reply:
Reusable component under review in #227